tox.ini

```diff
@@ -38,6 +38,8 @@ deps =
   sphinx
   -r{toxinidir}/docs/requirements-docs.txt
 commands = make -C docs html latex
+whitelist_externals =
+  make
```

Comment:
Ah, I see -- I had missed the fact that `make` was being used in the command. Either way is fine.